From e905a27ab2807ccd00ea290307c9842e7b293a9b Mon Sep 17 00:00:00 2001
From: leiwei <quic_leiwei@quicinc.com>
Date: Wed, 27 Dec 2023 16:02:42 +0800
Subject: [PATCH 375/500] rtnetlink: fix rtnetlink warning call trace

fix warning call trace caused by below openwrt patch:
I7a745b1e28cfc73f03e6d8a1f9d52c51dbde844c

Change-Id: I84afb46fd89fa55098bf572f70a5fe70b7f8e51d
Signed-off-by: leiwei <quic_leiwei@quicinc.com>
---
 net/core/rtnetlink.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/core/rtnetlink.c b/net/core/rtnetlink.c
index de231cfe7a8e..a0cc11103f26 100644
--- a/net/core/rtnetlink.c
+++ b/net/core/rtnetlink.c
@@ -61,7 +61,7 @@
 #include "dev.h"
 
 #define RTNL_MAX_TYPE		50
-#define RTNL_SLAVE_MAX_TYPE	44
+#define RTNL_SLAVE_MAX_TYPE	45
 
 struct rtnl_link {
 	rtnl_doit_func		doit;
-- 
2.34.1

